app/views/badges/index.html.erb

```diff
@@ -0,0 +1,62 @@
+<div class="container main servers-show">
+	<div class="row content">
+		<div class="col-sm-12">
+			<div class="palette report">
+				<h1>Server <span style="color: #B74C18">Badges</span></h1>
```

Comment:
I know it's kind of a pain, but could you move these inline styles into a css file?